src/HealthChecks.AzureServiceBus/AzureEventHubHealthCheck.cs

```diff
@@ -62,12 +65,12 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
 
     private EventHubProducerClient CreateClient()
     {
-        if (_connectionString is not null)
-            return new EventHubProducerClient(_connectionString);
+        if (_options.ConnectionString is not null)
```

Comment:
Should `ConnectionString` take precedence over `Connection` object?

Reply:
Don't know if it matters.

Reply:
OK, let's keep it as is.